addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScannerUnitTest.java

```diff
@@ -127,20 +188,10 @@ public void shouldIntersectMultipleCspHeaders() throws HttpMalformedHeaderExcept
     }
 
     @Test
-    public void shouldAlertOnWildcardFrameAncestorsDirective() throws HttpMalformedHeaderException {
+    public void shouldAlertOnWildcardFrameAncestorsDirective() {
         // Given
-        HttpMessage msg = new HttpMessage();
-        msg.setRequestHeader("GET https://www.example.com/test/ HTTP/1.1");
-
-        msg.setResponseBody("<html></html>");
-        msg.setResponseHeader(
-                "HTTP/1.1 200 OK\r\n"
-                        + "Server: Apache-Coyote/1.1\r\n"
-                        + "Content-Security-Policy: frame-ancestors *; default-src 'self'\r\n"
-                        + "Content-Type: text/html;charset=ISO-8859-1\r\n"
-                        + "Content-Length: "
-                        + msg.getResponseBody().length()
-                        + "\r\n");
+        HttpMessage msg =
+                createHttpMessage("", "frame-ancestors *; default-src 'self'; form-action 'none'");
```

Comment:
Why not call `createHttpMessage(String)`? (Same comment for some of the following tests.)